app/build.gradle

```diff
@@ -216,6 +216,7 @@ dependencies {
     implementation "com.github.bumptech.glide:okhttp3-integration:$glideVersion"
     kapt("com.github.bumptech.glide:compiler:$glideVersion")
 
+    implementation 'com.ibm.icu:icu4j:70.1'
```

Comment:
We would rather not import a whole new dependency just for this single feature. We can just use `android.icu.text.CompactDecimalFormat`, and make it conditional for API 24+.

Reply:
I've fixed it.